site/install-debian.xml

```diff
@@ -104,8 +104,10 @@ limitations under the License.
         <h3>Using rabbitmq.com APT Repository</h3>
 	<ol>
 	  <li>
-	    Add the following line to your <code>/etc/apt/sources.list</code>:
-	    <pre class="sourcecode">deb http://www.rabbitmq.com/debian/ testing main</pre>
+	    Execute the following command to add the APT repository to your <code>/etc/apt/sources.list.d</code>:
+	    <pre class="sourcecode">
+            echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list
```

Comment:
@dastergon it is fine. We [do it differently](https://github.com/rabbitmq/rabbitmq-website/blob/master/site/api-guide.xml#L110) in most places but for this guide you don't have to worry about that.